src/main/java/application/baseball/Judgement.java

```diff
@@ -0,0 +1,25 @@
+package application.baseball;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judgement {
+    public int getTotalCount(List<Integer> computer, List<Integer> inputNumber) {
```

Comment:
```suggestion
    private int getTotalCount(List<Integer> computer, List<Integer> inputNumber) {
```
내부에서만 사용되는 메서드네요. 그렇다면 접근제한자를 private을 통해 외부로 노출되지 않도록 하는 것이 좋을 것 같습니다.